docs/userdoc/Building.md

```diff
@@ -5,10 +5,10 @@
 * a CMake generator target (`make` is tested)
 * a C++17 compiler (gcc11, clang13, and ~~icpc 2019~~ are tested)
 * OpenMP (comes with GCC, for Clang you need `libomp`)
-* For rule based tuning: `pkg-config`
 
 Optional:
 * For `tuningLogToSQL`: `libsqlite3`
+* For rule based tuning: `pkg-config`. Rule based tuning can be disabled via the CMake (see below).
```

Comment:
```suggestion
* For rule based tuning: `pkg-config`. Rule based tuning can be disabled via the CMake (see [below](#disabling-rule-based-tuning)).
```

as it is not directly below

Reply:
This does not work with doxygen - my local doxygen (1.10.0) just does not include the link and the CI's doxygen build causes the CI to fail. Do you know that this should work with doxygen or do you know how to fix this?

Reply:
Okay issue fixed in `0529d104`. I had to use a different way of linking